src/binding.cc

```diff
@@ -139,6 +139,34 @@ void Print(const v8::FunctionCallbackInfo<v8::Value>& args) {
   fflush(stdout);
 }
 
+static v8::Local<v8::Uint8Array> ImportBuf(v8::Isolate* isolate, deno_buf buf) {
+  auto ab = v8::ArrayBuffer::New(
+      isolate, reinterpret_cast<void*>(buf.alloc_ptr), buf.alloc_len,
+      v8::ArrayBufferCreationMode::kInternalized);
+  auto view =
+      v8::Uint8Array::New(ab, buf.data_ptr - buf.alloc_ptr, buf.data_len);
+  return view;
+}
+
+static deno_buf ExportBuf(v8::Isolate* isolate,
+                          v8::Local<v8::ArrayBufferView> view) {
+  auto ab = view->Buffer();
+  auto contents = ab->Externalize();
+
+  deno_buf buf;
+  buf.alloc_ptr = reinterpret_cast<uint8_t*>(contents.Data());
+  buf.alloc_len = contents.ByteLength();
+  buf.data_ptr = buf.alloc_ptr + view->ByteOffset();
+  buf.data_len = view->ByteLength();
+
+  // Prevent JS from modifying buffer contents after exporting.
+  ab->Neuter();
```

Comment:
Should we add a test to demo that JS cannot indeed modify the buffer after neutered (after deno.send)?

Reply:
Yeah I agree.